curvefs/src/metaserver/inode_storage.cpp

```diff
@@ -185,9 +185,74 @@ MetaStatusCode InodeStorage::Insert(const Inode& inode, int64_t logIndex) {
     return MetaStatusCode::STORAGE_INTERNAL_ERROR;
 }
 
+MetaStatusCode InodeStorage::UpdateDeletingKey(
+  const Inode& inode, int64_t logIndex) {
+    WriteLockGuard lg(rwLock_);
+    Key4Inode key(inode.fsid(), inode.inodeid());
```

Comment:
Here you can add a new struct like `Key4DeletedInode` in which you can add deleted prefix before inode key